src/utils/version.js

```diff
@@ -31,9 +32,16 @@ const version = {
       libPromise.then((data) => {
         // Update version allowed in redux
         store.dispatch(isVersionAllowedUpdate({allowed: hathorLib.helpers.isVersionAllowed(data.version, hathorLib.constants.MIN_API_VERSION)}));
-        // Update network in redux
-        store.dispatch(networkUpdate({network: data.network}));
-        resolve();
+        // Set network in lib to use the correct address byte
+        let network;
+        if (data.network === 'mainnet') {
+          network = 'mainnet';
+        } else {
+          // Can we assume it will be testnet? I think it's safe
```

Comment:
I think so. We can define that all other networks are testnet except the mainnet.

Reply:
Agreed